test/runtests.jl

```diff
@@ -171,9 +171,17 @@ end
 
 # Recursing into submodules
 
+# should not be included in the result
```

Comment:
It'd be nice to explain how it is tested exactly. I guess, it's the numbers in `AgStats`. But this is not very explicit. Best would be to take the result of `is_stable_module`, which is an array of method checks, so it has methods in it, and check that `ext` is not in that array. But at least, referring from the comment to `AgStats` could make it more clear that it is tested somehow...